content/05.administrative.md

```diff
@@ -5,6 +5,8 @@ This work was funded through the Alex’s Lemonade Stand Foundation (ALSF) Child
 This project has been funded in part with Federal funds from the National Cancer Institute, National Institutes of Health, under Contract No. 75N91019D00024, Task Order No. 75N91020F00003 (JLR, ACR, APH). 
 The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products or organizations imply endorsement by the U.S. Government.
 
+The authors would like to thank the following collaborators who contributed to or supervised contributions to the analysis repository, but did not qualify for authorship: William Amadio, Holly Beale, Ellen Kephart, Alfred Lyle, and Olena Vaske.
```

Comment:
```suggestion
The authors would like to thank the following collaborators who contributed to, or supervised contributions to, the analysis repository: William Amadio, Holly Beale, Ellen Kephart, Alfred Lyle, and Olena Vaske.
```

I would probably prefer to take out the "did not qualify" clause since it sounds a bit negative.

Reply:
To @sjspielman's point I did originally have something in about according to guidelines. I'm happy to put that back in, which might make it seem less negative.

As a reader, I would wonder why folks who contributed to the analysis repository did not get granted authorship. (And the answer is that they did not meet the guidelines.)

Reply:
What about something along the lines of contributed analyses to the OpenPBTA project but didn't end up in the manuscript? (Not loving that either but takes out the did not qualify piece and turns it more positive)